docs/en/Get-Started.md

```diff
@@ -25,13 +25,17 @@ The optional sections will be labeled with **[Bonus]**.
 **Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
 If you are trying to speedup SQL analytics, you can try the
 [Presto Alluxio Getting Started](https://www.alluxio.io/alluxio-presto-sandbox-docker/) tutorial.
+<br />
```

Comment:
although a few additional line breaks may look nice for now, but i don't think it's a practice that can be set as a guideline that others will follow. it's a multi-way battle of "making it look more presentable" + "maintaining consistency  across all docs by defining guidelines" + "enforcing the guidelines for all future changes".

i also don't want to encourage people to use html within the markdown.